pages/contact.js

```diff
@@ -6,67 +7,98 @@ import Row from 'react-bootstrap/Row';
-                                        <Form.Group controlId="emailInput">
-                                            <Form.Label>Email address</Form.Label>
-                                            <Form.Control
-                                                name="email"
-                                                type="email"
-                                                placeholder="john.smith@mail.me"
-                                                required
-                                            />
-                                        </Form.Group>
-                                        <Form.Group controlId="messageTextarea">
-                                            <Form.Label>Message</Form.Label>
-                                            <Form.Control
-                                                name="message"
-                                                as="textarea"
-                                                rows="5"
-                                                placeholder="Go ahead, we're listening..."
-                                                required
-                                            />
-                                        </Form.Group>
-                                        <Button variant="outline-info" type="submit">
-                                            Submit
-                                        </Button>
-                                    </Form>
-                                </Card.Text>
-                            </Card.Body>
-                        </Card>
-                    </Col>
-                </Row>
-            </Container>
-        </div>
-    );
+        this.state = {
+            isAuthenticated: false,
+            name: '',
+        };
+    }
+
+    componentDidMount = () => {
+        const authToken = window.localStorage.getItem('auth_token');
+
+        if (!authToken) {
+            this.setState({ isAuthenticated: false });
+        } else {
+            axios.post(`${host}:${port}/authstatus`, { headers: { 'Authorization' : `Bearer ${authToken}` } }).then(res => {
+                if (res.status === 200) {
+                    this.setState({ isAuthenticated: true, name: res.data.authData.name });
+                }
+            }).catch(err => {});
+        }
+    }
+
+    render() {
+        const { isAuthenticated, name } = this.state;
+
+        return (
+            <div style={{ backgroundImage: `url(${Background})`, maxWidth: '100%', height: '100%' }}>
+                <link
+                    rel="stylesheet"
+                    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+                    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+                    crossorigin="anonymous"
+                />
+
+                <NavbarBootstrap isAuthenticated={isAuthenticated} name={name} />
```

Comment:
the auth state is only used to pre-populate the navbar based on whether or not user is authed
if authed, then shows the user icon with name, etc
if unauthed, then shows signup/login options